actions/branch-name-to-hostname-label/main.test.ts

```diff
@@ -0,0 +1,36 @@
+import {branchNameToHostnameLabel} from './main'
+jest.mock('../utils')
+jest.mock('@actions/core')
+
+// reset the counter on mock fn calls after every test
+beforeEach(() => jest.clearAllMocks())
+
+describe(`Branch Name to Hostname Label Action`, () => {
+    test(`sanitizes a full git ref into a valid hostname label`, async () => {
```

Comment:
Nitpick: personally I prefer `it` as it reads better than `test`. But very much a stylistic comment. :)